docker.sh

```diff
@@ -1,17 +1,24 @@
-#!/bin/bash -e
+#!/bin/bash
+
+DOCKER_NAME='smartcourse-testing'
+
+docker kill $DOCKER_NAME
+docker rm $DOCKER_NAME
+
+set -ex
 
 docker pull microsoft/mssql-server-linux:2017-latest
```

Comment:
Ummm i'm not sure if this is needed for travis or w/e, but if we're running this script locally we don't want it re-downloading the image every time, so maybe it should check the output of `docker image ls | grep mssql | grep 2017-latest` before downloading?

Reply:
Was slightly lazy; I need @PapaPwn to write it better. It's a simple if statement.

Reply:
Also I think if docker is running it cache's the image.

Reply:
Here:
```if sudo docker image ls | grep mssql | grep -q 2017-latest; then
  docker pull microsoft/mssql-server-linux:2017-latest
fi```
Actually i'll just test this and push it to this branch

Reply:
So I think you can just delete this line @DarkPurple141